common/models/role.js

```diff
@@ -210,26 +210,42 @@ module.exports = function(Role) {
         if (callback) callback(null, matches(ownerId, userId));
         return;
       } else {
```

Comment:
@pierreclr Hi, for some reason I'm no longer maintaining this pr, it's glad to see somebody else fulfilled the request for mult-owner. Hope to see this be merged into the matser soon.